theme_buzzy/views/new_page_template.xml

```diff
@@ -165,12 +165,6 @@
     </xpath>
 </template>
 
-<template id="new_page_template_about_s_text_cover" inherit_id="website.new_page_template_about_s_text_cover">
```

Comment:
Here again, let's not remove templates for now.
Replace the xpath with something that makes sense for this snippet.